backend/model/model.go

```diff
@@ -734,7 +749,7 @@ type Session struct {
 	// The version of Highlight's Firstload.
 	FirstloadVersion string `json:"firstload_version"`
 	// The client configuration that the end-user sets up. This is used for debugging purposes.
-	ClientConfig *string `json:"client_config" sql:"type:jsonb"`
+	ClientConfig *string `json:"client_config" sql:"type:jsonb" gorm:"type:jsonb"`
```

Comment:
is the `sql` tag still necessary?

Reply:
probably not, looks like we only use the `sql:` tag for these `jsonb` fields so I'll try removing them all